recipes/MEDIA/ASR/seq2seq/hparams/train_with_wav2vec.yaml

```diff
@@ -0,0 +1,156 @@
+# ################################
+# Model: Wav2Vec + DNN + CTC + Softmax
+# Authors modifier: Gaelle Laperriere
+# ################################
+
+# ------ Paths and parameters
+
+# Seed needs to be set at top of yaml, before objects with parameters are made.
+seed: 4242
+__set_seed: !!python/object/apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/media_ASR_wav2vec/<seed>
+cer_file_test: !ref <output_folder>/cer_test.txt
+ctc_file_test: !ref <output_folder>/ctc_test.txt
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+original_data_folder: !PLACEHOLDER # Path of folders S0272 and E0024.
+skip_wav: False # Skip the wav files storing if already done before.
+method: Null # Remove or keep specifiers in concepts. Method used by default.
+task: asr # Parse SLU or ASR data.
+skip_prep: True # Skip data preparation to csv because already done.
+
+# See https://github.com/pytorch/fairseq/blob/main/examples/wav2vec/README.md
+# for Wav2Vec models and https://huggingface.co/LeBenchmark for French ones.
+wav2vec_url: LeBenchmark/wav2vec2-FR-3K-large
+
+# Data files:
+data_folder: !ref <save_folder>/csv
+csv_train: !ref <data_folder>/train.csv
+csv_valid: !ref <data_folder>/dev.csv
+csv_test: !ref <data_folder>/test.csv
+
+# Data parameters:
+# With data_parallel batch_size is split into N jobs.
+# With DDP batch_size is multiplied by N jobs.
+batch_size: 4
+test_batch_size: 2
+# We remove utterances longer than 90s in the train/dev/test sets as
+# longer sentences certainly correspond to "open microphones".
+avoid_if_longer_than: 90.0
+avoid_if_smaller_than: 0.0
+dataloader_options:
+    batch_size: !ref <batch_size>
+    num_workers: 3
+    shuffle: True
+test_dataloader_options:
+    batch_size: !ref <test_batch_size>
+    num_workers: 3
+
+# Feature parameters:
+sample_rate: 16000
+feats_dim: 1024
+
+# Training parameters:
+number_of_epochs: 100
+lr: 1
+lr_wav2vec: 0.0001
+annealing_factor: 0.8
+annealing_factor_wav2vec: 0.9
+improvement_threshold: 0.0025
+improvement_threshold_wav2vec: 0.0025
+patient: 0
+patient_wav2vec: 0
+sorting: ascending
+
+# Model parameters:
+activation: !name:torch.nn.LeakyReLU
+dnn_blocks: 3
+dnn_neurons: 512
+
+# Wav2Vec parameters:
+freeze: False
+
+# Decoding parameters:
+blank_index: 0
+
+# Outputs:
+output_neurons: 159
+
+# ------ Functions and classes
+
+epoch_counter: !new:speechbrain.utils.epoch_loop.EpochCounter
+    limit: !ref <number_of_epochs>
+
+wav2vec: !new:speechbrain.lobes.models.huggingface_wav2vec.HuggingFaceWav2Vec2
+    source: !ref <wav2vec_url>
+    output_norm: True
+    freeze: !ref <freeze>
+    save_path: !ref <save_folder>/wav2vec.pt
+
+dec: !new:speechbrain.lobes.models.VanillaNN.VanillaNN
```

Comment:
this is an enc not a dec (we called it like that here https://github.com/speechbrain/speechbrain/blob/c075b41a85a71a0fb74edaec28c2a202e98f9234/recipes/LibriSpeech/ASR/CTC/hparams/train_hf_wav2vec.yaml and in many other recipes)